manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -276,6 +280,55 @@ def modify_vserver(self, vserver_name, aggregate_names, retention_hours):
         }
         self.send_request('vserver-modify', modify_args)
 
+    @na_utils.trace
+    def _modify_security_cert_expire_days(self, vserver_name,
```

Comment:
I suggest to go with a more general name `_modify_security_cert()` , expire-days is one of potentially many options this function may have in the future

Reply:
fixed